src/combinations.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+'use strict';
+/**
+ * Implementation of Combinations
```

Comment:
Could this give a more informative summary of the functionality, like the [existing functions](http://simplestatistics.org/docs/)? While it's good to link to Wikipedia we want simple-statistics to be informative without needing to follow links. Something like:

> Combinations are unique subsets of a collection - in this case, `k` elements from a collection at a time.